### DISCUSSION TITLE
Transfer of PBPK model between versions

### DISCUSSION THREAD

**Discussion by nina-nauwelaerts** [Open-Systems-Pharmacology/Forum#1609] [Questions & Problems]
Hi, 

I would like to transfer all the PBPK models I'm working on in version 9 of PK Sim to version 11.0. Therefore, I exported the models to snapshot, and loaded them in v11.0. 

For most medicines, this resulted in very similar results when I compared the predictions. However, for metformin the predicted plasma concentration is shifted downwards and the total plasma clearance is higher. In version 9, I have been working further on the model for metformin as developed by Nina Hanke. I already noticed that the Vmax of some transporters was not taken over when uploading the snapshot in version 11, but even after correcting this there is a difference between the versions. When I compare the AUC and Cmax for each simulation between version 9 and 11, there seems to be an equal shift downwards in all simulations (see figure where each point represents one simulation). 
![plot](https://github.com/Open-Systems-Pharmacology/Forum/assets/63354539/bbfa9cd5-d7a8-40a5-9025-a0bb2d6a8fdf)
I'm wondering what the reason for this difference is? Is there a difference between the versions that can explain this observation, or should I look further for parameters that were not taken over correctly (so far I did not observe any differences)? 

In addition, I also noticed that the 'number of repetitions' and 'time between repetitions' in the advanced administration protocol were not always taken over correctly when I used snapshot to transfer PBPK models between versions. I'm not sure if this issue has been fixed/reported already?

**Comment by wilbertdew** [comment]
Hi @nina-nauwelaerts,

A first thing you might want to check is the (intracellular) concentration of the involved enzymes.
My understanding is that the calculation method of expression levels has been changed between these versions.
Some information is provided here for the v10.0 release: https://github.com/Open-Systems-Pharmacology/Suite/releases
Others might be able to provide more detail on this. A related discussion is located here: https://github.com/Open-Systems-Pharmacology/Forum/discussions/1151
Hope you can find the cause of this discrepancy!
